lua/lspconfig/server_configurations/awk_language_server.lua

```diff
@@ -0,0 +1,27 @@
+local util = require 'lspconfig.util'
+
+return {
+  default_config = {
+    cmd = { 'awk-language-server' },
+    filetypes = { 'awk' },
+    single_file_support = true,
+    handlers = {
```

Comment:
This can be removed now that https://github.com/neovim/neovim/pull/17149#event-5930028005 is merged. I would recommend gating the configuration behind a 0.7 check (you can refer to lemminx for how to do this)

Reply:
@mjlbach I can't find the version gating within lemminx.lua

Reply:
Ah, sorry vdmj https://github.com/neovim/nvim-lspconfig/pull/1094/files#diff-cc4d28b8a44a7645573450364c61477dffef8cf48b0005630bfaed16ab150b2aR4-R7

Reply:
gated version to >=0.7 using `vim.version()` and removed `handlers` in https://github.com/neovim/nvim-lspconfig/pull/1665/commits/721ae1d51d9dac7a4eef05a989d2ae31e05614ed